doc_source/CloudWatch-Events-Input-Transformer-Tutorial.md

```diff
@@ -58,4 +58,4 @@ We achieve this by mapping the *instance* variable to the `$.detail.state` JSON
 
 1. Choose **Configure details**\.
 
-1. Type a name and description for the rule, and choose **Create rule**\.
\ No newline at end of file
+1. Type a name and description for the rule, and choose **Create rule**\.
```

Comment:
No idea why this line is showing as modified.

Reply:
Looks like you added a trailing newline character.

Reply:
I see. I used GitHub's direct file edit feature, so I blame that. :-)

Reply:
Thanks, mhurne, good catch.